app/controllers/administrate/application_controller.rb

```diff
@@ -111,6 +108,18 @@ def find_resource(param)
       resource_class.find(param)
     end
 
+    def resource_includes
+      association_classes = [
+        Administrate::Field::HasMany, Administrate::Field::HasOne,
+        Administrate::Field::BelongsTo
+      ]
+
+      dashboard.class::ATTRIBUTE_TYPES.map do |key, value|
+        key if association_classes.include?(value) ||
+               association_classes.include?(value.try :deferred_class)
```

Comment:
This feels a little messy. Maybe we can split the checking here into one line, the `try` to another?